### PR TITLE
Update the header of every file - new comments

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/107129
It's related to the task above.
Like previous pull request, feel free to add or change the statements.

---

Don't merge this pr until the label `In progress` will be removed.

### CODE REVIEW

**Comment by slavaleleka** [comment]
I suppose we should mention that types of rules should be similar for single annoyance while scroll unblocking (for example)

File: AnnoyancesFilter/sections/cookies_general.txt [hunk 1]
```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of the rules are against cookies. Rules should be generic.
```
Comment (stanislav-atr) [inline AnnoyancesFilter/sections/cookies_general.txt:2]: I guess 'cookie notifications' would be more appropriate. Same for the `cookies specific` file.

File: AnnoyancesFilter/sections/cookies_whitelist.txt [hunk 2]
```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of the rules that fixes incorrect blocking. Rules should be specific.
```
Comment (stanislav-atr) [inline AnnoyancesFilter/sections/cookies_whitelist.txt:2]: Shouldn't it be specified as 'incorrect blocking of cookie notifications'? Might be misleading as it is now.

File: AnnoyancesFilter/sections/cookies_general.txt (hunk 1 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_general.txt:2]: Yep, I can change it.

File: AnnoyancesFilter/sections/cookies_whitelist.txt (hunk 2 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_whitelist.txt:2]: I can also change it, but it's kinda a clear point that the user add fixes to the `cookie_whitelist.txt`.

File: AnnoyancesFilter/sections/annoyances.txt [hunk 3]
```diff
@@ -1,4 +1,10 @@
 !
+! This section contains the list of the rules are against annoyances that can't be blocked in other files. Rules should be specific.
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt:2]: `can't be blocked in other files` => `not falling under existing categories`. We can block literally all in this file.

File: AnnoyancesFilter/sections/antiadblock.txt [hunk 4]
```diff
@@ -1,6 +1,13 @@
...
-! These rules are against all kinds of ad blocking reinjection/circumvention scripts
-! which does not affect the functioning of site
+! This section contains the list of the rules are against all kinds of ad blocking reinjection/circumvention scripts which does not affect the functioning of site.
+! Also the section has common rules that can help prevent most known detections. For instance, BlockAdBlock (bab). In that case you should add domain to the common rules.
+! Rules should be specific, but in some cases it's allowed to add generic ones.
+! Note, that you can use any type of the rules, except ones from the general_url.txt, adservers.txt, whitelist_stealth.txt.
+!
+! Good: ||example.org/adblock/detect.js
+! Good: @@/adblocker/detect.js$domain=example.org
+! Bad: ||ad.doubleclick.net^ (should be in adservers.txt)
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/antiadblock.txt:9]: **Bad** example) Sometimes anti-adblokс scripts are loaded from third-party sources.

File: AnnoyancesFilter/sections/antiadblock.txt [hunk 5]
```diff
@@ -1,6 +1,13 @@
 !
-! These rules are against all kinds of ad blocking reinjection/circumvention scripts
-! which does not affect the functioning of site
+! This section contains the list of the rules are against all kinds of ad blocking reinjection/circumvention scripts which does not affect the functioning of site.
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/antiadblock.txt:2]: First of all - anti-adblocks(move to the first line).

File: AnnoyancesFilter/sections/antiadblock.txt [hunk 6]
```diff
@@ -1,6 +1,13 @@
...
+! Also the section has common rules that can help prevent most known detections. For instance, BlockAdBlock (bab). In that case you should add domain to the common rules.
+! Rules should be specific, but in some cases it's allowed to add generic ones.
+! Note, that you can use any type of the rules, except ones from the general_url.txt, adservers.txt, whitelist_stealth.txt.
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/antiadblock.txt:5]: I think this line is redundant.

File: AnnoyancesFilter/sections/cookies_general.txt [hunk 7]
```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of the rules that block cookie notifications cookie notifications. Rules should be generic.
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_general.txt:2]: `This section contains the list of the rules...` => `This section contains the list of generic rules...`

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 8]
```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of the rules that block cookie notifications. Rules should be specific.
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_specific.txt:2]: `This section contains the list of the rules` => `This section contains the list of [domain]specific rules`

File: AnnoyancesFilter/sections/css_extended.txt [hunk 9]
```diff
@@ -1,5 +1,9 @@
 !
-! Extended CSS rules
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/css_extended.txt:2]: Better mark this file as obsolete. I want to remove it after sorting.

File: AnnoyancesFilter/sections/low-attendance.txt [hunk 10]
```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of the rules are against ads, however websites have less than 30k visits per month. Rules should be specific.
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/low-attendance.txt:2]: Not even 30k. Literally near zero attendance.

File: AnnoyancesFilter/sections/popups.txt [hunk 11]
```diff
@@ -1,8 +1,9 @@
-!#######################################################
-!######### Rules for blocking in page popups ###########
-!######### and floating elements #######################
-!#######################################################
-! Whitelist is at the end of file
+!
+! This section contains the list of the rules are against popup windows (popup) and floating elements. Whitelist is at the end of file
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt:2]: `popup windows` => `in page popups`

File: AnnoyancesFilter/sections/remains.txt [hunk 12]
```diff
@@ -1,5 +1,9 @@
 !
-!######### Remains of ads. Obsolete section. Better use Extended CSS rules in regular filters
+! This is obsolete section contains the list of the rules that block remains of ads. Better use Extended CSS rules in regular filters
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/remains.txt:2]: Add `Do not add any rules here`.

File: AnnoyancesFilter/sections/self-promo.txt [hunk 13]
```diff
@@ -1,3 +1,10 @@
+!
+! This is section contains the list of the rules that block self promotion.
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/self-promo.txt:2]: promotion**s** or `own site banners`

File: AnnoyancesFilter/sections/tweaks.txt [hunk 14]
```diff
@@ -1,3 +1,10 @@
+!
+! This is section contains the list of the rules that is used for CSS tweaks. For instance, we unblock right context menu, disabled selection of the text - things that are not related to ads.
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/tweaks.txt:2]: `CSS tweaks` - not only.
`things that are not related to ads` redundant and obviously in this filter)

File: AnnoyancesFilter/sections/widgets.txt [hunk 15]
```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of the rules that block useless widgets (weather, money exchange, 3rd-party feedbacks, 3rd-party assistants).
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/widgets.txt:2]: `useless`  better `annoying`)

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/antiadblock.txt (hunk 4 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/antiadblock.txt:9]: I know, however this example is for beginners - adserver should be in adservers file :)

File: MobileFilter/sections/general_url.txt [hunk 16]
```diff
@@ -1,5 +1,9 @@
 !
-! General URL(for apps only)
+! This section contains the list of the base rules are against ads (for apps only). Rules should be generic.
```
Comment (Alex-302) [inline MobileFilter/sections/general_url.txt:2]: `for apps only` Not apps only. Just for mobile sites(99%) or apps.

File: MobileFilter/sections/specific_app.txt [hunk 17]
```diff
@@ -1,5 +1,8 @@
 !
-! Section contains rules for specific applications
+! This section contains the list of the base rules are against ads (for apps only). Rules should be specific.
```
Comment (Alex-302) [inline MobileFilter/sections/specific_app.txt:2]: `are against ads (for apps only)` => `are against ads in mobile apps`

File: SpywareFilter/sections/mobile_whitelist.txt [hunk 18]
```diff
@@ -2,6 +2,12 @@
 ! Exception rules that fix broken mobile apps and mobile websites
```
Comment (Alex-302) [inline SpywareFilter/sections/mobile_whitelist.txt:2]: `! Exception rules that fix broken mobile apps and mobile websites`
old line

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/antiadblock.txt [hunk 19]
```diff
@@ -1,6 +1,12 @@
 !
-! These rules are against all kinds of ad blocking reinjection/circumvention scripts
-! which does not affect the functioning of site
+! First of all, this section has common rules that can help prevent most known detections. For instance, BlockAdBlock (bab). In that case you should add domain to the common rules.
```
[the remark this answers is not in the document]
Comment (zloyden) [reply AnnoyancesFilter/sections/antiadblock.txt:2]: Well, what should I mention here? I can also remove this - we have examples at the beginning of the file.
```
For instance, BlockAdBlock (bab). In that case you should add domain to the common rules.
```

File: ChineseFilter/sections/general_extensions.txt [hunk 20]
```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of the rules are against ads.
+! Rules should be specific and you can use only some type of the rules: css, scriptlets, JS, HTML with `$$` syntax.
```
Comment (ameshkov) [inline ChineseFilter/sections/general_extensions.txt:3]:
```suggestion
! By advanced we mean the following rule types: CSS, scriptlets, HTML filtering (`$$` marker), JS.
```

File: ChineseFilter/sections/general_url.txt [hunk 21]
```diff
@@ -1,4 +1,10 @@
 !
+! This section contains the list of the generic rules against ads.
```
Comment (ameshkov) [inline ChineseFilter/sections/general_url.txt:2]:
```suggestion
! This section contains the list of generic blocking rules.
```

File: ChineseFilter/sections/general_url.txt [hunk 22]
```diff
@@ -1,4 +1,10 @@
...
+!
+! Good: .org/ads/
+! Bad: .org/ads/$domain=example.org (for instance, should be in general_extensions.txt)
```
Comment (ameshkov) [inline ChineseFilter/sections/general_url.txt:5]: Why `general_extensions.txt`? This is a domain-specific rule.

**Review by ameshkov — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 23]
```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of specific rules that block cookie notifications.
```
[the remark this answers is not in the document]
Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_specific.txt:2]: `domain-specific` in all related files

File: AnnoyancesFilter/sections/css_extended.txt [hunk 24]
```diff
@@ -1,5 +1,11 @@
...
+! Obsolete. Do not add new rules.
```
[the remark this answers is not in the document]
Comment (Alex-302) [reply AnnoyancesFilter/sections/css_extended.txt:2]: We will remove it recently.

File: AnnoyancesFilter/sections/remains.txt [hunk 25]
```diff
@@ -1,5 +1,9 @@
 !
-!######### Remains of ads. Obsolete section. Better use Extended CSS rules in regular filters
+! Obsolete. Do not add new rules.
```
[the remark this answers is not in the document]
Comment (Alex-302) [reply AnnoyancesFilter/sections/remains.txt:2]: Also will be removed. Leftovers on the sites with low attendance.

File: AnnoyancesFilter/sections/low-attendance.txt [hunk 26]
```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of the rules that block ads, however websites have literally zero attendance. Rules should be specific.
```
[the remark this answers is not in the document]
Comment (zloyden) [reply AnnoyancesFilter/sections/low-attendance.txt:2]: Did you mean two exceptions (probably anti-adblock case) or the whole file? Anyway, it's better to ask @Alex-302.

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 27]
```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of the rules are against banners of mobile apps for the sites.
+! Note, that we have additional sections that have different purposes. For instance, we can add exceptions or limit the rules for some platforms.
```
[the remark this answers is not in the document]
Comment (zloyden) [reply AnnoyancesFilter/sections/mobile-app.txt:3]: We block elements that contain an offer (suggestion) to install a mobile application.

> we can add exceptions or limit the rules for some platforms.

The point is that we have different sections in the file and users have to look there before adding any rules.

File: AnnoyancesFilter/sections/popups.txt [hunk 28]
```diff
@@ -1,8 +1,9 @@
-!#######################################################
-!######### Rules for blocking in page popups ###########
-!######### and floating elements #######################
-!#######################################################
-! Whitelist is at the end of file
+!
+! This section contains the list of the rules that block in page popups and floating elements. Whitelist is at the end of file.
```
[the remark this answers is not in the document]
Comment (zloyden) [reply AnnoyancesFilter/sections/popups.txt:2]: Ok, will add some examples.

File: ChineseFilter/sections/general_url.txt (hunk 22 above)
Comment (zloyden) [reply ChineseFilter/sections/general_url.txt:5]: Indeed.

File: ChineseFilter/sections/antiadblock.txt [hunk 29]
```diff
@@ -1,5 +1,11 @@
 !
-! These rules are against all kinds of ad blocking reinjection/circumvention scripts
+! First of all, this section has common rules that can help prevent most known detections. For instance, BlockAdBlock (bab). In that case you should add domain to the common rules.
+! The section contains the list of the rules are against all kinds of ad blocking reinjection/circumvention scripts.
```
[the remark this answers is not in the document]
Comment (zloyden) [reply ChineseFilter/sections/antiadblock.txt:3]: > that do not affect the functioning of the website.

Actually, this is related to the AdGuard Annoyances.

File: AnnoyancesFilter/sections/mobile-app.txt (hunk 27 above)
[the remark this answers is not in the document]
Comment (ameshkov) [reply AnnoyancesFilter/sections/mobile-app.txt:3]:
```suggestion
! Note, that the file consists of multiple sections for different purposes. For instance, there is a section for exceptions and some platform-specific sections.
```

File: ChineseFilter/sections/antiadblock.txt (hunk 29 above)
[the remark this answers is not in the document]
Comment (ameshkov) [reply ChineseFilter/sections/antiadblock.txt:3]: Yeah, that's right, updated suggestion

File: DutchFilter/sections/whitelist.txt [hunk 30]
```diff
@@ -1,5 +1,9 @@
 !
-! White list. Fixing filtration errors (false-positive)
+! This section contains the list of the rules that fix incorrect blocking. Rules should be specific.
```
Comment (ameshkov) [inline DutchFilter/sections/whitelist.txt:2]:
```suggestion
! This section contains the list of rules that fix incorrect blocking. Rules must be domain-specific.
```

File: EnglishFilter/sections/cname_adservers.txt [hunk 31]
```diff
@@ -1,5 +1,9 @@
 !
-! CNAME advertising networks
+! This section contains the list of the rules are against CNAME advertising networks.
+!
+! Good: !#include https://raw.githubusercontent.com/AdguardTeam/cname-trackers/master/trackers/adspyglass.txt
```
Comment (ameshkov) [inline EnglishFilter/sections/cname_adservers.txt:4]: It should be explained that we only include rules from AdGuard's cname-tracker repo here, there's no need to add regular rules here.

File: EnglishFilter/sections/content_blocker.txt [hunk 32]
```diff
@@ -1,6 +1,11 @@
 !
-! Section contains list of the rules for content blockers
-! It will be used by AdGuard Content Blocker and AdGuard for iOS/Safari
+! This section contains the list of the rules for content blockers (AdGuard for iOS, AdGuard for Safari and AdGuard Content Blocker for Yandex Browser / Samsung Browser on Android).
```
Comment (ameshkov) [inline EnglishFilter/sections/content_blocker.txt:2]:
```suggestion
! This section contains the list of the rules for content blockers (AdGuard for iOS, AdGuard for Safari and AdGuard Content Blocker for Yandex Browser / Samsung Browser on Android).
!
! The point is that these apps support only a limited subset of AdGuard filtering rules syntax. 
! Because of that, we have to resort to using more complicated and less effective solutions there.
```

File: EnglishFilter/sections/cryptominers.txt [hunk 33]
```diff
@@ -1,5 +1,9 @@
 !
-! Section contains list of crypto-miners
+! This section contains the list of the rules are against crypto-miners.
```
Comment (ameshkov) [inline EnglishFilter/sections/cryptominers.txt:2]:
```suggestion
! This section contains the list of rules are against crypto-miners.
```

File: EnglishFilter/sections/foreign.txt [hunk 34]
```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of the rules are against ads from sites where the language doesn't match other regional filters.
```
Comment (ameshkov) [inline EnglishFilter/sections/foreign.txt:2]:
```suggestion
! This section contains the list of rules that are supposed to work on websites we have no language-specific filter for yet.
```

File: EnglishFilter/sections/replace.txt [hunk 35]
```diff
@@ -1,5 +1,9 @@
 !
-! The content replacement rules
+! This section contains the list of the content replacement rules that block ads. Rules should be specific.
```
Comment (ameshkov) [inline EnglishFilter/sections/replace.txt:2]:
```suggestion
! This section contains the list of the content replacement rules that block ads. Rules must be domain-specific.
```

File: EnglishFilter/sections/whitelist.txt [hunk 36]
```diff
@@ -1,5 +1,9 @@
 !
-! White list. Fixing filtration errors (false-positive)
+! This section contains the list of the rules that fix incorrect blocking. Rules should be specific.
```
Comment (ameshkov) [inline EnglishFilter/sections/whitelist.txt:2]:
```suggestion
! This section contains the list of rules that fix incorrect blocking. Rules should be domain-specific.

```

File: EnglishFilter/sections/whitelist_stealth.txt [hunk 37]
```diff
@@ -1,6 +1,10 @@
-!-----------------------------------------------------------
-!-------------------- Fixing Stealth Mode ------------------
-!-----------------------------------------------------------
+!
+! This section contains the list of the base rules that fixes Stealth Mode incorrect blocking. Rules should be specific.
```
Comment (ameshkov) [inline EnglishFilter/sections/whitelist_stealth.txt:2]:
```suggestion
! This section contains the list of rules that fix Stealth Mode incorrect blocking. Rules should be domain-specific.
```

**Review by ameshkov — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/cname_adservers.txt (hunk 31 above)
Comment (zloyden) [reply EnglishFilter/sections/cname_adservers.txt:5]: Added a note about it.

File: UsefulAdsFilter/sections/usefulads.txt [hunk 38]
```diff
@@ -1,3 +1,11 @@
+!
+! This section contains the list of the base rules that unblock search enginge result pages. Rules should be specific.
```
Comment (ameshkov) [inline UsefulAdsFilter/sections/usefulads.txt:2]:
```suggestion
! This section contains the list of the rules that unblock contextual (and only contextual) ads on the search engine result pages. Rules should be domain-specific.

```

File: TurkishFilter/sections/specific.txt [hunk 39]
```diff
@@ -1,4 +1,11 @@
 !
+!
+! This section contains the list of domain-specific rules that block ads.
+!
+! Good: example.org###rek
```
Comment (ameshkov) [inline TurkishFilter/sections/specific.txt:5]: Basic rules are also allowed in all `specific.txt` files, should be added to examples.

File: TurkishFilter/sections/general_url.txt [hunk 40]
```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of generic blocking rules.
```
Comment (ameshkov) [inline TurkishFilter/sections/general_url.txt:2]: I've updated this in some other lists, could you please copy/paste it here?

File: TurkishFilter/sections/css_extended.txt [hunk 41]
```diff
@@ -1,5 +1,9 @@
 !
-! Extended CSS rules
+! This section contains the list of the extended CSS rules against ads.
```
Comment (ameshkov) [inline TurkishFilter/sections/css_extended.txt:2]:
```suggestion
! This section contains the list of extended CSS rules against ads.
```

File: TrackParamFilter/sections/whitelist.txt [hunk 42]
```diff
@@ -1,5 +1,9 @@
 !
-! Fixing filtering errors
+! This section contains the list of the rules that fix incorrect blocking. Rules should be specific and with only one modifier `removeparam`.
```
Comment (ameshkov) [inline TrackParamFilter/sections/whitelist.txt:2]:
```suggestion
! This section contains the list of rules that fix incorrect blocking. Rules should be domain-specific and only the `removeparam` modifier is allowed.
```

File: TrackParamFilter/sections/specific.txt [hunk 43]
```diff
@@ -1,6 +1,9 @@
 !
-! The rules for specific sites
+! This section contains the list of domain-specific rules that remove url tracking parameters. Rules have to contain only one modifier `removeparam`.
```
Comment (ameshkov) [inline TrackParamFilter/sections/specific.txt:2]:
```suggestion
! This section contains the list of domain-specific rules that remove URL tracking parameters. Rules have to contain only a single modifier: `removeparam`.
```

File: TrackParamFilter/sections/general_url.txt [hunk 44]
```diff
@@ -1,5 +1,9 @@
 !
-! General rules
+! This section contains the list of generic blocking rules.
```
Comment (ameshkov) [inline TrackParamFilter/sections/general_url.txt:2]:
```suggestion
! This section contains the list of generic `$removeparam` rules that are applied on every website.
! You should only put here known parameters that are used on many websites.
! IMPORTANT: add a comment that explains where this parameter comes from (what tracking company is using it).
! Ideally, supply a link to the document that explains how this parameter is used.
```

**Review by ameshkov — CHANGES_REQUESTED** [review]
Some more changes. Also, check out the comments there.

**Comment by urvbcn88** [comment]
No need for all these explanations. I'm sure a lot of people wouldn't even notice or care.

File: TurkishFilter/sections/general_url.txt (hunk 40 above)
Comment (zloyden) [reply TurkishFilter/sections/general_url.txt:2]: > ! This section contains the list of generic blocking rules.

I updated this text in all related files.

File: TurkishFilter/sections/specific.txt (hunk 39 above)
Comment (zloyden) [reply TurkishFilter/sections/specific.txt]: Added.

File: AnnoyancesFilter/sections/subscriptions.txt [hunk 45]
```diff
@@ -1,6 +1,10 @@
-!#######################################################
-!######### Subscriptions, newsletters etc. #############
-!#######################################################
+!
+! This is section contains the list of rules that block the promotion of subscriptions, newsletters.
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/subscriptions.txt:2]: `This is section contains` => `This section contains`

File: ChineseFilter/sections/antiadblock.txt [hunk 46]
```diff
@@ -1,5 +1,11 @@
 !
-! These rules are against all kinds of ad blocking reinjection/circumvention scripts
+! First of all, this section has common rules that can help prevent most known detections. For instance, BlockAdBlock (bab). In that case you should add domain to the common rules.
```
Comment (Alex-302) [inline ChineseFilter/sections/antiadblock.txt:2]: `you should add` better `should be added`

File: EnglishFilter/sections/whitelist_stealth.txt [hunk 47]
```diff
@@ -1,6 +1,10 @@
-!-----------------------------------------------------------
-!-------------------- Fixing Stealth Mode ------------------
-!-----------------------------------------------------------
+!
+! This section contains the list of rules that fix Stealth Mode incorrect blocking. Rules should be domain-specific.
```
Comment (Alex-302) [inline EnglishFilter/sections/whitelist_stealth.txt:2]: `incorrect blocking` better `issues`. Not all options are blocking.

File: SocialFilter/sections/general_elemhide.txt [hunk 48]
```diff
@@ -1,5 +1,9 @@
 !
-! Section contains list general hiding rules
+! This section contains generic element hiding rules that block social widgets.
+!
+! Good: ##.adv
```
Comment (Alex-302) [inline SocialFilter/sections/general_elemhide.txt:4]: better social filter related example

File: SpywareFilter/sections/general_elemhide.txt [hunk 49]
```diff
@@ -1,5 +1,9 @@
 !
-! Hide counters images
+! This section contains generic element hiding rules that block counters images.
```
Comment (Alex-302) [inline SpywareFilter/sections/general_elemhide.txt:2]: `counters images` not images only, also popular leftovers.

File: TrackParamFilter/sections/specific.txt [hunk 50]
```diff
@@ -1,6 +1,9 @@
 !
-! The rules for specific sites
+! This section contains the list of domain-specific rules that remove URL tracking parameters. Rules have to contain only a single modifier: `removeparam`.
```
Comment (Alex-302) [inline TrackParamFilter/sections/specific.txt:2]: " only a single modifier: `removeparam`."
content type and `domain`  are also modifiers)

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: ChineseFilter/sections/antiadblock.txt [hunk 51]
```diff
@@ -1,5 +1,11 @@
 !
-! These rules are against all kinds of ad blocking reinjection/circumvention scripts
+! First of all, this section has common rules that can help prevent most known detections. In that case domain should be added to the common rules.
```
Comment (Alex-302) [inline ChineseFilter/sections/antiadblock.txt:2]: ` that can help prevent most known detections.` => ` that prevent most known ad block detections.` or something like that.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/general_elemhide.txt [hunk 52]
```diff
@@ -1,5 +1,9 @@
 !
-! Hide counters images
+! This section contains generic element hiding rules that block counters images, leftovers.
```
Comment (Alex-302) [inline SpywareFilter/sections/general_elemhide.txt:2]: `counters images` blocked by url. Leftovers only. It makes no sense to hide counters that are not blocked.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]
